tutorials/14-food-review-tone-analysis-and-response.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<p>Can we load ChatGPT UDF by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/georgia-tech-db/evadb/pull/991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
It turns out to be already part of bootstrap. I will delete it.